twill-yarn/src/main/java/org/apache/twill/internal/appmaster/ApplicationMasterService.java

```diff
@@ -38,7 +38,6 @@
 import com.google.common.util.concurrent.SettableFuture;
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
```

Comment:
Please don't update import spaces. If you use IntelliJ or eclipse, you can download the style from the twill site.